.github/workflows/cargo-doc.yaml

```diff
@@ -20,7 +20,11 @@ jobs:
         with:
           toolchain: nightly-2023-07-13
           override: true
-
+      - name: Install Protoc
```

Comment:
do we really need to ask informal to make this a dependency?

Reply:
We can't build Hermes importing `namada_sdk` and `namada` without `protoc`.